cmd/nerdctl/run.go

```diff
@@ -132,7 +132,7 @@ func setCreateFlags(cmd *cobra.Command) {
 	// dns is defined as StringSlice, not StringArray, to allow specifying "--dns=1.1.1.1,8.8.8.8" (compatible with Podman)
 	cmd.Flags().StringSlice("dns", nil, "Set custom DNS servers")
 	cmd.Flags().StringSlice("dns-search", nil, "Set custom DNS search domains")
-	cmd.Flags().StringSlice("dns-opt", nil, "Set DNS options")
+	cmd.Flags().StringSlice("dns-option", nil, "Set DNS options")
```

Comment:
https://github.com/docker/cli/blob/05d288eb06f2653149c0aa16e9ac455da9f64e21/cli/command/container/opts.go#L213-L218

Can we support `dns-opt` too?
Can be another PR though

Reply:
SGTM, done.